tests/validations/mta/mtaValidations.spec.ts

```diff
@@ -0,0 +1,284 @@
+import {
+  Disposable,
+  mockFileSystemWatcher,
+  mockVscode,
+  testVscode,
+} from "../../mockUtil";
+mockVscode("src/validations/mta/mtaValidations");
+import * as sinon from "sinon";
+import { expect } from "chai";
+import {
+  DEV_MTA_EXT,
+  MTA_YAML,
+  updateMtaDiagnostics,
+  validateWsMtaYamls,
+  watchMtaYamlAndDevExtFiles,
+} from "../../../src/validations/mta/mtaValidations";
+import { Diagnostic, Range, Uri } from "vscode";
+import * as mtaDiagnostic from "../../../src/validations/mta/mtaDiagnostic";
+import { resolve } from "path";
+import * as fsExtra from "fs-extra";
+import { mta, Mta } from "@sap/mta-lib";
+
+describe("mtaValidations", () => {
+  const disposables: Disposable[] = [];
+
+  beforeEach(() => {
+    mtaDiagnostic.clearCurrentDiagnosticCollections();
+  });
+
+  afterEach(() => {
+    sinon.restore();
+  });
+
+  describe("watchMtaYamlAndDevExtFiles", async () => {
+    it("registers to workspace.createFileSystemWatcher and to events with the correct parameters", async () => {
```

Comment:
The test name should reflect what is expected instead of how it is expected to be done
```suggestion
    it("registers to changes in mta.yaml and dev.mtaext files in the workspace", async () => {
```